src/utils/api/main.js

```diff
@@ -13,4 +13,14 @@ mainApi.getMemoRoomList = async (userId) => {
   return response.data;
 };
 
+mainApi.postNewMemoRoom = async ({ userId, name }) => {
+	const response = await axios.post(
+    `/users/${userId}/memorroms/new`,
+    { name },
+    { withCredentials: true }
+    );
+
+	return response.data;
```

Comment:
```suggestion
	const response = await axios.post(
    `/users/${userId}/memorroms/new`,
    { name },
    { withCredentials: true }
    );

	return response.data;
```
인덴팅이 조금 이상한것 같아요!! 확인 부탁드릴게요

Reply:
넵 감사합니다!!